src/fault-detection/models/classification.py

```diff
@@ -0,0 +1,74 @@
+import os
+from pathlib import Path
+import numpy as np
+import pandas as pd
+import pickle
+import joblib
+from dotenv import load_dotenv
+from keras.models import load_model
+
+
+# BASE_DIR = Path(__file__).resolve().parent
+# dotenv_path = os.path.join(BASE_DIR, '.env')
+load_dotenv()
+
+LSTM_MODEL = os.getenv('LSTM_MODEL','jobs/lstm_model.h5')
+SCALER = os.getenv('SCALER','jobs/scaler.pkl')
+
+def calibrate(list_of_dict):    
+        df = pd.DataFrame(list_of_dict)
+
+        #     if (not device_id or not s1_pm2_5 or not s2_pm2_5):
+        #         return jsonify({"message": "Please specify the device_id, datetime, sensor1 pm2.5, sensor2 pm2.5, sensor1 pm10, sensor1 pm10, temperature and humidity values in the body. Refer to the API documentation for details.", "success": False}), 400
+        #     calibrated_pm2_5, calibrated_pm10 = model.compute_calibrated_val(s1_pm2_5,s2_pm2_5)           
+        #     response.append({'device_id': device_id, 'calibrated_PM2.5': calibrated_pm2_5, 'calibrated_PM10': calibrated_pm10 })
+        # return jsonify(response), 200
```

Comment:
Hi @fajemila , this function doesn't seem to have a purpose. You might need to cleanup your code, remove commented out code blocks and unused functions